webapp/apps/taxbrain/helpers.py

```diff
@@ -1,14 +1,52 @@
 from collections import namedtuple
 import taxcalc
-import dropq
```

Comment:
Hmm.. when I tried this out, I needed to restore this line so that I could use dropq later in the file.